recipes/csv-parser/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CsvParserConan(ConanFile):
+    name = "csv-parser"
```

Comment:
I would namespace the name. `csv-parser` is very generic.
I propose to name it `vincentlaucsb-csv-parser`.

Reply:
Done